SPECS/sm.spec

```diff
@@ -11,7 +11,7 @@
 Summary: sm - XCP storage managers
 Name:    sm
 Version: 2.30.8
-Release: %{?xsrel}.1%{?dist}
+Release: %{?xsrel}.1.0.fix202.1%{?dist}
```

Comment:
Why this tag? We will release this change for everyone.
No difference with the previous release, it's just a new patch.